### PR TITLE
Dịch "lr-scheduler_vn" - Phần 5

### LINKED ISSUE #1996: Dịch "lr-scheduler_vn" - Phần 5
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_optimization/lr-scheduler_vn.md#L317-L368

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by davidnvq**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Điền số issue sau dấu `#` (không có dấu cách): Close #1996 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn. Bạn đã kiểm tra rằng mỗi đoạn văn đã nằm một dòng?

* [x] Điền tên đầy đủ của bạn vào mục **Những người thực hiện** ở cuối tập tin.

* [ ] Điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Lưu ý cho người phản biện (Reviewer)

1. Chỉ review nếu Pull Request (PR) này đã sẵn sàng cho việc review. Một PR sẵn sàng khi PR đó không có label `status: wip`.
2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

### CODE REVIEW

File: chapter_optimization/lr-scheduler_vn.md [hunk 1]
```diff
@@ -320,7 +320,7 @@ train(net, train_iter, test_iter, num_epochs, loss, trainer, ctx)
 ### Cosine Scheduler
 -->
 
-### *dịch tiêu đề phía trên*
+### Định thời Cô-sin
```
Comment (cuongvng) [inline chapter_optimization/lr-scheduler_vn.md:323]:
```suggestion
### Định thời Cô-sin
```
#2061

File: chapter_optimization/lr-scheduler_vn.md [hunk 2]
```diff
@@ -356,7 +360,8 @@ In the context of computer vision this schedule *can* lead to improved results.
 Note, though, that such improvements are not guaranteed (as can be seen below).
 -->
 
-*dịch đoạn phía trên*
+Trong ngữ cảnh thị giác máy, định thời này *có thể* cải thiện kết quả thu được.
```
[the remark this answers is not in the document]
Comment (davidnvq) [reply chapter_optimization/lr-scheduler_vn.md:363]: computer vision a nhớ k nhầm dịch là thị giác máy. Trường anh dạy luôn môn này ;)

[the remark this answers is not in the document]
Comment (cuongvng) [reply chapter_optimization/lr-scheduler_vn.md:363]: em thấy trong glossary là `thị giác máy tính`, `máy` nghe giống `machine vision`.

[the remark this answers is not in the document]
Comment (davidnvq) [reply chapter_optimization/lr-scheduler_vn.md:363]: a cũng k ý kiến đặc biệt về từ này. Nếu mn đã chọn từ trên thì lấy từ đó cho nhất quán vs phần trc.   a vừa check lại thì đúng là tên môn đó và tên lab đó là thị giác máy :))

File: chapter_optimization/lr-scheduler_vn.md [hunk 3]
```diff
@@ -320,7 +320,7 @@ train(net, train_iter, test_iter, num_epochs, loss, trainer, ctx)
 ### Cosine Scheduler
 -->
 
-### *dịch tiêu đề phía trên*
+### Định thời Côsin
```
Comment (duythanhvn) [inline chapter_optimization/lr-scheduler_vn.md:323]:
```suggestion
### Định thời Cô-sin
```
